### PR TITLE
slowdown LargeMessageClusterSpec for tcp transport, #24576

### PR DESCRIPTION
**Pull request by patriknw**
I have run this in the jenkins repeat job and it works fine.

Refs #24576

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
LGTM, I guess.

Could it be a useful idea to provide configurable throttling for the large message path through artery? Given that if this can happen in a test it can happen in an app, or is this entirely synthetic?

File: akka-cluster/src/multi-jvm/scala/akka/cluster/LargeMessageClusterSpec.scala [hunk 1]
```diff
@@ -44,20 +53,29 @@ object LargeMessageClusterMultiJvmSpec extends MultiNodeConfig {
...
-          compression {
-            #actor-refs.advertisement-interval = 2 second
-            #manifests.advertisement-interval = 2 second
-          }
         }
       }
     }
     """))
 
+  final case class Big(payload: Array[Byte])
```
Comment (ktoso) [inline akka-cluster/src/multi-jvm/scala/akka/cluster/LargeMessageClusterSpec.scala]: Maybe Slow expresses the intent more?

**Review by ktoso — APPROVED** [review]

File: akka-cluster/src/multi-jvm/scala/akka/cluster/LargeMessageClusterSpec.scala (hunk 1 above)
Comment (patriknw) [reply akka-cluster/src/multi-jvm/scala/akka/cluster/LargeMessageClusterSpec.scala]: ok

**Comment by patriknw** [comment]
Throttling is an interesting idea, it would be on the byte level so it's not just slapping the throttle stage there. We also want to promote the stream refs for such stuff when we have implemented the bytes bulk transfer.

**Comment by patriknw** [comment]
Renamed Big to Slow, will merge after validation

**Pull request merged**